lib/fmrest/spyke/json_parser.rb

```diff
@@ -191,11 +193,7 @@ def delete_request?(env)
       # @param source [String] a JSON string
       # @return [Hash] the parsed JSON
       def parse_json(source)
-        if defined?(::MultiJson)
-          ::MultiJson.load(source, symbolize_keys: true)
-        else
-          ::JSON.parse(source, symbolize_names: true)
-        end
+        JSON.parse(source, symbolize_names: true)
```

Comment:
there's no naming conflict for `JSON`, right? So do we need `::JSON` here? I'm fine to leave it in, if that's standard convention.

Reply:
The `::` is there just to ease my paranoia of "what if we later on create a module named `JSON` inside the `FmRest` namespace?", which of course wouldn't really happen in practice.